docs/security/host-certs.md

```diff
@@ -60,20 +60,64 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Generate a Certificate Signing Request (CSR) and private key:
 
+1. Create a custom openssl config file for this, `incommon.conf`, using your institution's information for the
```

Comment:
Are the CSR details intended to be sub-bullets of `1. Generate a Certificate Signing Request (CSR) and private key:`? If they are, the indentation is wrong